README.md

```diff
@@ -127,6 +127,9 @@ cargo install --locked cargo-pgrx
 
 **Important:** `cargo-pgrx` **must** be built using the same compiler as you'll use to build the rest of your project. Every time you update your Rust toolchain you have to also manually reinstall `cargo-pgrx`. See [Upgrading](#Upgrading), below.
 
```

Comment:
This part is no longer true on the develop line. Someone should probably fix that.

Reply:
oh? is cargo-pgrx no longer implemented as a rustc_driver?

Reply:
Never was! you're thinking of plrustc, which is subtly differently deranged.